geonode/settings.py

```diff
@@ -1749,6 +1749,7 @@ def get_geonode_catalogue_service():
 
 SOCIALACCOUNT_ADAPTER = 'geonode.people.adapters.SocialAccountAdapter'
 SOCIALACCOUNT_AUTO_SIGNUP = False
+ACCOUNT_GEONODE_LOCAL_SIGNUP = strtobool(os.environ.get('ACCOUNT_GEONODE_LOCAL_SIGNUP', 'True'))
```

Comment:
I would add a comment to exlain the rationale under this variable. It is useful in case we only want the social account signup enabled.

Reply:
done

Reply:
please let's chacnge the variable to "SOCIALACCOUNT_WITH_GEONODE_LOCAL_SINGUP"

Reply:
done